docs/usecases/client_server.md

```diff
@@ -0,0 +1,6 @@
+---
+id: usecases_client_server
+title:  "Client Server"
+---
+
+TODO remake existing `channel.md`
```

Comment:
So what do we need here?

Reply:
I've moved here the existing `tut/channel.md`, split it into functions, added comment for each part and provided example output. Also I've removed the old one.